tests/tests/test_demo_artifact.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
+# Copyright 2019 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        https://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+from MenderAPI import authentication, deployments, auth_v2
+from mendertesting import MenderTesting
+import subprocess
+import requests
+import signal
+import pytest
+import logging
+import conftest
+import os
+
+# NOTE - The password is set on a per test-basis, as it is generated on the fly by
+# the demo script.
+auth = authentication.Authentication(
+    username='mender-demo', email='mender-demo@example.com')
+authv2 = auth_v2.auth = auth
+deploy = deployments.Deployments(auth, authv2)
+
+
+# @pytest.mark.incremental
```

Comment:
For now, the tests do not pass when run simultaneously. Any good solutions to this? I've read pytest has some dependency solutions. Thoughts?

Reply:
Why is that? Doesn't the namespace keep them separate? Or is it the downloaded file which is the problem?

Reply:
fixed in https://github.com/mendersoftware/integration/pull/607/commits/943d0d996b9ed48886213c7f95c87699af7e6e92